source/tyr/tests/integration/ridesharing_service_test.py

```diff
@@ -0,0 +1,233 @@
+# coding: utf-8
+# Copyright (c) 2001-2018, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# channel `#navitia` on riot https://riot.im/app/#/room/#navitia:matrix.org
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from __future__ import absolute_import, print_function, division, unicode_literals
+from tyr import app
+from navitiacommon import models
+from tests.check_utils import api_get, api_delete, api_put
+import pytest
+import ujson
+
+
+@pytest.fixture
+def default_ridesharing_service_config():
+    with app.app_context():
+        ridesharing0 = models.RidesharingService('TestCovoiturage0')
+        ridesharing0.klass = 'jormungandr.scenarios.ridesharing.instant_system.InstantSystem'
+        ridesharing0.args = {
+            'service_url': 'http://bob.com',
+            "rating_scale_min": 0,
+            "crowfly_radius": 500,
+            "rating_scale_max": 5,
+            "api_key": "aaa",
+            "network": "Test Covoiturage",
+            "feed_publisher": {
+                "url": "url",
+                "id": "test-coverage",
+                "license": "Private",
+                "name": "Test ABC - TEST",
+            },
+        }
+        ridesharing1 = models.RidesharingService('TestCovoiturage1')
```

Comment:
can we create ridesharing1 with different parameter values than ridesharing0 so that we can check the difference.

Reply:
ok